app/serializers/benchmark_serializer.rb

```diff
@@ -3,9 +3,12 @@
 # JSON API serialization for an OpenSCAP Benchmark
 class BenchmarkSerializer
   include FastJsonapi::ObjectSerializer
+
   attributes :ref_id, :title, :version, :description, :os_major_version,
              :latest_supported_os_minor_versions
-  has_many :rules
+  has_many :rules do |benchmark, params|
+    benchmark.rules.paginate(per_page: params[:limit], page: params[:offset])
+  end
```

Comment:
We can make this DRY by either creating a wrapper around `has_many`, e.g. `has_many_pages_of` :scream_cat: or by just extracting this into a method.